packages/form-core/src/FieldApi.ts

```diff
@@ -1,8 +1,13 @@
-import type { DeepKeys, DeepValue, Updater } from './utils'
-import type { FormApi, ValidationError } from './FormApi'
+import {
+  isNonEmptyArray,
+  type DeepKeys,
+  type DeepValue,
+  type Updater,
+} from './utils'
+import type { FormApi, ValidationError, ValidationErrorMap } from './FormApi'
 import { Store } from '@tanstack/store'
 
-export type ValidationCause = 'change' | 'blur' | 'submit'
+export type ValidationCause = 'change' | 'blur' | 'submit' | 'mount'
```

Comment:
Glad you made this change - I intended to refactor our mount validation to include this